db/structure.sql

```diff
@@ -2568,14 +2632,14 @@ CREATE INDEX index_projects_on_migrated ON projects USING btree (migrated) WHERE
 
 
 --
--- Name: index_projects_on_mobile_friendly; Type: INDEX; Schema: public; Owner: -
+-- Name: index_projects_on_mobile_friendly; Type: INDEX; Schema: public; Owner: -; Tablespace: 
```

Comment:
Happened when I updated the structure, presumably because I had already updated my database from the master branch and a more recent migration. Should merge fine.